eksconfig/metrics-server.go

```diff
@@ -0,0 +1,18 @@
+package eksconfig
+
+// MetricsServerSpec defines the spec for the Addon
```

Comment:
I'm not sure if you noticed, but the way I wrote this comment makes it ultra copy-pastable :P